pkg/internal/checkup/vmi.go

```diff
@@ -143,10 +143,12 @@ func CloudInit(username, password string, bootCommands []string) string {
 
 func trafficGenBootCommands(configDiskSerial string) []string {
 	const configMountDirectory = "/mnt/app-config"
+	const testScriptsDirectory = "/opt/tests"
```

Comment:
this const is going to be used in the executrr  (see main branch reference [here](https://github.com/kiagnose/kubevirt-dpdk-checkup/blob/630d0497b2ba17a91245b93e37719ce50be4ddaa/pkg/internal/checkup/executor/trex/client.go#L123))
Do we want to make it a const on the trex package perhaps?

Reply:
This is a good idea.
The problem is, that currently there is no file with a proper name in the trex package to hold this const,
and I find creating a whole file just for this little const problematic.

I suggest to move it in the PR that will introduce the trex client logic.
WDYT?

Reply:
I agree.